src/handler/CompanyHandler.ts

```diff
@@ -0,0 +1,26 @@
+import API from "../API"
+import { getAPIKeyIDFromAttr } from "../helper"
+
+const COTTER_COMPANY_CACHE_EXPIRY_KEY = "cotter_company_cache_expiry"
+const COTTER_COMPANY_CACHE_KEY = "cotter_company_info"
+
+class CompanyHandler {
+  static getInfo(): any {
+    let companyInfo = localStorage.getItem(COTTER_COMPANY_CACHE_KEY)
```

Comment:
What's the company cache key for?

Reply:
So need to store companyInfo in the localStorage, so if it's still not expired, we can use the one in localStorage then update it on the background.